Simplenote/Classes/SPOnboardingViewController.swift

```diff
@@ -99,15 +111,19 @@ private extension SPOnboardingViewController {
         }
     }
 
-//    func presentAuthenticationInterface(mode: AuthenticationMode) {
-//        guard let simperiumAuthenticator = authenticator else {
-//            fatalError()
-//        }
-//
-//        let controller = SPAuthHandler(simperiumService: simperiumAuthenticator)
-//        let viewController = SPAuthViewController(controller: controller, mode: mode)
-//        navigationController?.pushViewController(viewController, animated: true)
-//    }
+    func ensureNavigationBarIsHidden() {
+        navigationController?.setNavigationBarHidden(true, animated: true)
+    }
+
+    func presentAuthenticationInterface(mode: AuthenticationMode) {
+        guard let simperiumAuthenticator = authenticator else {
+            fatalError()
```

Comment:
Maybe provide a description? Or log the issue before calling `fatalError`?

Reply:
Added a description! Thank you!!